vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/ApiSteps.java

```diff
@@ -229,6 +219,16 @@ public void setCustomRequestConfig(ExamplesTable configItems) throws ReflectiveO
         httpTestContext.putRequestConfig(requestConfigBuilder.build());
     }
 
+    private HttpResponse issueRequest(HttpMethod httpMethod, String url, String relativeURL) throws IOException
+    {
+        HttpResponse response = httpRequestExecutor.executeHttpCallSafely(httpMethod, url, relativeURL);
+        if (response != null)
+        {
+            httpTestContext.releaseRequestData();
```

Comment:
actually it should be wrapped in `try-finally` to guarantee exception handling, the same for case in `JsonResponseValidationSteps`, also please avoid code duplication